lambdas/sharedattributes/src/test/java/uk/gov/di/ipv/core/sharedattributes/SharedAttributesHandlerTest.java

```diff
@@ -107,10 +141,10 @@ void shouldExtractSessionIdFromHeaderAndReturnSharedAttributesAndStatusOK()
         when(userIdentityService.getUserIssuedCredentials(SESSION_ID))
                 .thenReturn(
                         Map.of(
+                                "CredentialIssuer",
+                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_2),
                                 "CredentialIssuer1",
-                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_1),
-                                "CredentialIssuer2",
-                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_2)));
+                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_1)));
```

Comment:
It's a bit confusing that `CREDENTIAL_INPUT_2` is first in this map - maybe rename it's key to `CredentialIssuer2` and re-order them?